auth/api/iam/api.go

```diff
@@ -384,8 +387,7 @@ func (r Wrapper) handleAuthorizeRequest(ctx context.Context, ownDID did.DID, req
 // RFC9101: The OAuth 2.0 Authorization Framework: JWT-Secured Authorization Request (JAR).
 func (r Wrapper) RequestJWTByGet(ctx context.Context, request RequestJWTByGetRequestObject) (RequestJWTByGetResponseObject, error) {
 	ro := new(jarRequest)
-	// TODO: burn request object to prevent DoS through signing requests https://github.com/nuts-foundation/nuts-node/issues/3063
-	err := r.authzRequestObjectStore().Get(request.Id, ro)
+	err := r.authzRequestObjectStore().GetAndDelete(request.Id, ro)
```

Comment:
is this principle mentioned in the RFC (request object can only be retrieved once)? Since from HTTP perspective, HTTP GET requests are non-mutative, meaning they should be executable multiple times without side effects, and without the response changing. My thought; could there be clients that try to retrieve it multiple times (for whatever reason)?

Reply:
the spec says it should exist short-ish, or once if it contains personal identifiable information. Considering this results in a signing operation I chose short-ish==once